### PR TITLE
Update helm instruction to use latest chart in Readme

### PR DESCRIPTION
**Pull request by developStorm**
## Summary

* Update helm instruction to use latest chart hosted in this repository

## Checklist

- [X] Categorize the PR by setting a good title and adding one of the labels:
      `kind:bug`, `kind:enhancement`, `kind:documentation`, `kind:change`, `kind:breaking`, `kind:dependency`
      as they show up in the changelog

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,13 @@ For more information about external storage in kubernetes, see
...
 ## Installation
 
-Recommended option is the [Helm Chart][helm chart].
+Recommended option is via Helm:
+
+```bash
+helm repo add kubernetes-zfs-provisioner https://ccremer.github.io/kubernetes-zfs-provisioner/
+helm repo update
+helm install kubernetes-zfs-provisioner zfs-provisioner/kubernetes-zfs-provisioner
+```
```
[the remark this answers is not in the document]
Comment (developStorm) [reply README.md]: Oh I didn't notice thres's a new readme available! In that case I think update the original helm chart link might be better?: [335a458](https://github.com/ccremer/kubernetes-zfs-provisioner/pull/73/commits/335a45898405f05b5fae29de2fd8d940b3b9da0a)

File: README.md [hunk 2]
```diff
@@ -22,7 +22,8 @@ For more information about external storage in kubernetes, see
 
 ## Installation
 
-Recommended option is the [Helm Chart][helm chart].
+Recommended option is via [Helm][helm chart]
+```
```
Comment (ccremer) [inline README.md:26]: I think that's a leftover?
```suggestion
```

**Review by ccremer — APPROVED** [review]
one last bit

File: README.md (hunk 2 above)
Comment (developStorm) [reply README.md]: Whoops let's squash merge this PR so this doesn't go into commit history 😅

**Comment by ccremer** [comment]
Thanks for the contribution!

**Pull request merged**